cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx

```diff
@@ -263,6 +295,76 @@ class OpenCVControlComponent extends React.PureComponent<Props & DispatchToProps
         }
     };
 
+    private onTracking = async (e: Event): Promise<void> => {
+        const {
+            isActivated, jobInstance, frame, curZOrder, fetchAnnotations,
+        } = this.props;
+
+        if (!isActivated) {
+            return;
+        }
+
+        const { activeLabelID, trackedShapes, activeTracker } = this.state;
+        const [label] = jobInstance.labels.filter((_label: any): boolean => _label.id === activeLabelID);
+
+        const { isDone, shapesUpdated } = (e as CustomEvent).detail;
+        if (!isDone || !shapesUpdated || !activeTracker) {
+            return;
+        }
+
+        try {
+            const { points } = (e as CustomEvent).detail.shapes[0];
+            const imageData = this.getCanvasImageData();
+            const trackerModel = activeTracker.model();
+            trackerModel.init(imageData, points);
+            const state = new core.classes.ObjectState({
+                shapeType: ShapeType.RECTANGLE,
+                objectType: ObjectType.TRACK,
+                zOrder: curZOrder,
+                label,
+                points,
+                frame,
+                occluded: false,
+                attributes: {},
+                descriptions: [`Trackable (${activeTracker.name})`],
+            });
+            const [clientID] = await jobInstance.annotations.put([state]);
+            this.setState({
+                trackedShapes: [
+                    ...trackedShapes,
+                    {
+                        clientID,
+                        trackerModel,
+                        shapePoints: points,
+                    },
+                ],
+            });
+
+            // update annotations on a canvas
+            fetchAnnotations();
+        } catch (err) {
+            notification.error({
+                description: err.toString(),
+                message: 'Tracking error occured',
+            });
+        }
+    };
+
+    private getCanvasImageData = ():ImageData => {
+        const canvas: HTMLCanvasElement | undefined = window.document.getElementById('cvat_canvas_background') as
```

Comment:
It returns element or null, not undefined. 

https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById

Reply:
Done